target/linux/ath79/image/generic.mk

```diff
@@ -722,6 +722,22 @@ define Device/engenius_ews511ap
 endef
 TARGET_DEVICES += engenius_ews511ap
 
+define Device/engenius_emr3000
+  SOC := qca9558
+  DEVICE_VENDOR := EnGenius
+  DEVICE_MODEL := EMR3000
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca988x-ct \
+	kmod-usb2 kmod-usb-storage
+  IMAGE_SIZE := 32064k
+  IMAGES += factory.bin
+  IMAGE/factory.bin := append-kernel | pad-to $$$$(BLOCKSIZE) | \
+	append-rootfs | pad-rootfs | check-size | \
+	senao-header -r 0x101 -p 0x83 -t 2
+  IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs | \
+	append-metadata
```

Comment:
Did you remove the pad-to for a reason?

Reply:
Thanks for comment. 
I just reference other models, but I don't think it any effect with firmware.
If it has special meaning?

Reply:
Is there something speaking against just using the default recipe in Device/Default node (image/Makefile), i.e. just drop your line here?

Reply:
Thanks for comment.
Drop the the line and using the default recipe.